lib/user/user-session-spec.js

```diff
@@ -0,0 +1,43 @@
+var mockMbaasServiceProxy = require('./stub/mbaas-service-proxy');
+var mediator = require('fh-wfm-mediator/lib/mediator');
+var express = require('express');
+var proxyquire = require('proxyquire');
+var sinon = require('sinon');
+var assert = require('assert');
+
+var sessionManager;
+
+describe("User Sessions", function() {
+  var verifySessionStub;
+
+  beforeEach(function() {
+    verifySessionStub = mockMbaasServiceProxy.getMockVerifySessionStub();
+    sessionManager = proxyquire('./user-session', {
+      './mbaas-service-proxy': mockMbaasServiceProxy.getMockSessionObject(verifySessionStub)
+    });
+    sessionManager(mediator, express(), 'my-service-name-guid');
+  });
+
+  it("Testing valid session token", function(done) {
+    mediator.request('wfm:user:session:validate', 'myvalidsessiontoken').then(function(validationResponse) {
```

Comment:
For tests with promise-based functions, just return the promise and leave out the `done` callback. If the promise resolves, then mocha passes the test. If it rejects, then the test fails.